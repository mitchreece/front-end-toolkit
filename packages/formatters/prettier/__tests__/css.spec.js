const { resolve } = require('path');
const { getFileContent, formatFile } = require('./utils/file');

const filePath = resolve(__dirname, './samples/sample.css');

describe('CSS formatting', () => {
    test('matches snapshot', () => {
        const fileContents = getFileContent(filePath);
        const formattedFile = formatFile(fileContents, { parser: 'css' });

        expect(formattedFile).toMatchSnapshot();
    });
});
