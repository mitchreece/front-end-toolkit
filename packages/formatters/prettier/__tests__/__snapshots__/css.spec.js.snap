// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CSS formatting matches snapshot 1`] = `
":root {
  --colour-lightest: #ffffff;
  --colour-light: #e6e6e6;
  --color-primary: #53f342;
  --color-error: red;
  --colour-font-light: #e6e6e6;
  --colour-font-dark: #333333;
}

html,
body {
  padding: 0;
  margin: 0;
  color: var(--colour-font-dark);
}

.card {
  background-color: var(--colour-lightest);
}

.button--cta {
  color: var(--colour-light);
  background-color: var(--colour-primary);
}

.button--error {
  color: var(--colour-light);
  background-color: var(--colour-error);
}
"
`;
